sambacc/commands/ctdb.py

```diff
@@ -257,3 +257,10 @@ def ctdb_must_have_node(ctx: Context) -> None:
             return
         _logger.info("node not yet ready")
         waiter.wait()
+
+
+@commands.command(name="ctdb-check-ok")
```

Comment:
We discussed in a private chat that this name could be improved. After a little more thinking I think we should mirror what is going on in `sambacc/commands/check.py` I suggest "ctdb-check" with a target value that is currently "node-status" (or "nodestatus" either is fine.)